doc/guides/writing-tests.md

```diff
@@ -60,11 +60,12 @@ the nature of the test requires that the test run without it.
 The second line loads the `common` module. The `common` module is a helper
 module that provides useful tools for the tests.
 
-Even if no functions or other properties exported by `common` are used in a
-test, the `common` module should still be included. This is because the `common`
-module includes code that will cause tests to fail if variables are leaked into
-the global space. In situations where no functions or other properties exported
-by `common` are used, it can be included without assigning it to an identifier:
+Even if a test uses no functions or other properties exported by `common`,
+the test should still include the `common` module. This is because the `common`
```

Comment:
How about `include the common module before all other modules.` as a reasonable concise compromise that we can expand on later if it is found to be inadequate?